dashboard/lib/tasks/seed.rake

```diff
@@ -650,7 +655,7 @@ namespace :seed do
     files_to_import.each {|file_to_import| CsvToSqlTable.new(pegasus_dir(file_to_import), db, table_prefix).import}
   end
 
-  FULL_SEED_TASKS = [:check_migrations, :videos, :concepts, :scripts, :courses, :reference_guides, :data_docs, :callouts, :school_districts, :schools, :census_summaries, :secret_words, :secret_pictures, :donors, :foorms, :import_pegasus_data, :datablock_storage].freeze
+  FULL_SEED_TASKS = [:check_migrations, :skills, :videos, :concepts, :scripts, :courses, :reference_guides, :data_docs, :callouts, :school_districts, :schools, :census_summaries, :secret_words, :secret_pictures, :donors, :foorms, :import_pegasus_data, :datablock_storage].freeze
```

Comment:
I would suggest moving this to `SCRIPT_DEPENDENCIES`, after `:check_migrations` and before `:deprecated_blockly_levels`. This will help prevent issues if for example you add a skill, add it to a level, and then someone runs `rake seed:scripts` without doing a full seed.